app/Http/Controllers/StaticController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Contracts\View\Factory;
+use Illuminate\Contracts\View\View;
+use Illuminate\Foundation\Application;
+use Illuminate\Http\Request;
+use App\Models\Course;
+use App\Models\Test;
+
+class StaticController extends Controller
+{
+    /**
+     * @return View
+     */
+    public function home()
+    {
+        return view("home");
+    }
+
+    /**
+     * @return View
+     */
+    public function profile()
+    {
+        return view("profile");
+    }
+
+    /**
+     * @return View
+     */
+    public function dashboard()
+    {
+        $courses = Course::all();
+        $ecsBehaald = 0;
```

Comment:
Dutch and English variable naming, which makes it messy to read. Choose one language (within your team) and use this consistently.